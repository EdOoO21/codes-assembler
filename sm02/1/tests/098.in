177 702 -912 -279 -144 -880 990 -31 -423 491 673 389 139 -868 614 17 863 -599 -186 -930 342 653 443 556 -261 -992 589 597 620 824 -985 541 -215 583 -377 -316 -906 314 526 425 776 -6 -606 702 686
