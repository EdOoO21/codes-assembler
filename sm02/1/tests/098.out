686
702
-606
-6
776
425
526
314
-906
-316
-377
583
-215
541
-985
824
620
597
589
-992
-261
556
443
653
342
-930
-186
-599
863
17
614
-868
139
389
673
491
-423
-31
990
-880
-144
-279
-912
702
177
